driver_test.go

```diff
@@ -1263,6 +1264,48 @@ func testArray(t *testing.T, json bool) {
 	})
 }
 
+func TestVector(t *testing.T) {
+	testVector(t, false)
+}
+
+func testVector(t *testing.T, json bool) {
```

Comment:
Instead of such approach, we use `t.Run` - please see some examples in our tests.

Reply:
Oh ok, I was following the example of `testArray` and `testVariant` which call `runDBTest` without `t.Run`. I've converted this test into a table test that uses `t.Run`.